.github/workflows/code.yml

```diff
@@ -8,12 +8,6 @@ on:
     types: [opened, synchronize, reopened, labeled]
     branches:
       - main
-    paths-ignore:
-      - "*.md"
-      - '*.yaml'
-      - '*.yml'
-      - LICENSE
-      - 'docs/**'
```

Comment:
We almost never push any changes which do not require at least mock & unit tests. Integration tests are already excluded to only run when a specific label is present.

Reply:
In SYSENG-867 we decided to only run them if we actually change code and not just docs but given that, as you said we almost exclusively only push changes that require tests, I think we could challenge the decision from back then.

Reply:
Good point. I still think we need to refine this workflow.. but I removed that change for now.